detect-configuration/src/main/java/com/blackducksoftware/integration/hub/detect/configuration/DetectConfigWrapper.java

```diff
@@ -147,6 +148,16 @@ public void setDetectProperty(final DetectProperty detectProperty, final String
         return propertyMap;
     }
 
+    public Map<String, String> getPropertyKeyMap() {
```

Comment:
This wont work as expected. `final Map<String, String> blackduckHubProperties = detectConfigWrapper.getPropertyKeyMap();
        hubServerConfigBuilder.setFromProperties(blackduckHubProperties);` That will not work the way we want because some of the properties we want to pass into hubServerConfigBuilder are not Strings, and this method is only getting the STRING properties

Reply:
at least blackduck.hub.timeout and blackduck.hub.trust.cert